pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -110,30 +110,36 @@ func statusConfigMapLabels(node string) map[string]string {
 	}
 }
 
-// UpdateNodeStatus updates the status ConfigMap for the OSD on the given node. It returns the name
+// UpdateNodeStatus updates the status ConfigMap for the OSD on the given node or pvc. It returns the name
 // the ConfigMap used.
-func UpdateNodeStatus(kv *k8sutil.ConfigMapKVStore, node string, status OrchestrationStatus) string {
-	labels := statusConfigMapLabels(node)
+func UpdateNodeStatus(kv *k8sutil.ConfigMapKVStore, nodeOrPVCName string, status OrchestrationStatus, isNode bool) string {
+	labels := statusConfigMapLabels(nodeOrPVCName)
 
 	// update the status map with the given status now
 	s, _ := json.Marshal(status)
-	cmName := statusConfigMapName(node)
+	cmName := statusConfigMapName(nodeOrPVCName)
 	if err := kv.SetValueWithLabels(
 		cmName,
 		orchestrationStatusKey,
 		string(s),
 		labels,
 	); err != nil {
+		var kind string
+		if isNode {
+			kind = "node"
+		} else {
+			kind = "PVC"
+		}
 		// log the error, but allow the orchestration to continue even if the status update failed
-		logger.Errorf("failed to set node %q status to %q for osd orchestration. %s", node, status.Status, status.Message)
+		logger.Errorf("failed to set %s %q status to %q for osd orchestration. %s", kind, nodeOrPVCName, status.Status, status.Message)
```

Comment:
The difference between node and pvc is hopefully obvious by the name of the node or pvc that will be printed. I'm not sure it's worth the complexity of all the changes in this PR just to clarify whether it is a node or pvc. What if the message just said that it is either a node or pvc like this? Whoever is debugging should know which one they're using. 
```suggestion
		logger.Errorf("failed to set node or pvc %q status to %q for osd orchestration. %s", node, status.Status, status.Message)
```
Or if you really found this change would be helpful, could you show an example of the message that was confusing?

Reply:
@travisn Thanks, I agree with you. Fixed.